foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -18,9 +18,9 @@ The `flex` declaration is actually a shorthand for 3 properties that you can set
 
 In this case, `flex` is actually a shorthand for `flex-grow`, `flex-shrink` and `flex-basis`.
 
-![flex shorthand](https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/10.png)
+![flex shorthand](https://cdn.statically.io/gh/Hamza-Aziane/top-curriculum/4ebd6ac6abdb09d95731f3975372193543af1e08/foundations/html_css/flexbox/imgs/10.png)
```

Comment:
@Hamza-Aziane 2 notes for you on this image :-)

1. I think we will need to use a different statically link here. Did you check out the images guide here? (https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum) The new image could be added as a separate PR (pull request). Then the statically link gets generated with that commit id. If you make a new PR by following that guide you can update this link here with the new generated link.

2. What do you think about cropping out some of the extra black space on the new image so it's a bit more centered?

@thatblindgeye Just wanted to let you know that the old flex image is going to be changed and it happens to be the example image you use in the Layout Style Guide. Not sure if this will have an impact on that and if you will want to update it or not! Also if you know any more info about the above with the statically links please let me know :-)

Reply:
Hi @dm-murphy, I think that I should start all over to be able to follow the **two PRs process**, I wasn't able to do it while keeping this PR. Any ideas?

Reply:
Hi @aziham 👋 

Do whatever would be easiest for you! If you'd like we can close this one down and you can start fresh. 

Otherwise you could probably make a new PR that just adds the image (ping me so I see it) and after that has been merged you could then use that commit ID to make the new statically link for this PR. 

But please do whatever would be easiest for yourself and thanks again for all your efforts here! 🙏

Reply:
> Otherwise you could probably make a new PR that just adds the image (ping me so I see it) and after that has been merged you could then use that commit ID to make the new statically link for this PR.

@dm-murphy Thanks for the solution.

🤦🏻‍♂️ I was thinking in a rigid way about this issue to the point I haven't considered modifying this PR after the one that adds the image is merged.

I usually tend to make my commit history as "perfect" as it can be, I think that's where I messed up. Thanks again Daniel for this mindset shifting answer.

I'll create a new branch for the image PR and ping you when it's done. Have a nice day.

Reply:
@dm-murphy, Hi Daniel.

I created a new branch that adds the image file, and submitted a PR for it: #24477

PS: I cropped the image as you suggested.

Reply:
Hi @aziham 👋 

Thanks for making the new PR for adding the image file, but looks like your branch there had some prior commits attached. If you take a look at the Commits and Files Changed tabs at the top you'll see some changes to the JS lessons too. 

No worries, but you will want to either clean up that PR by removing those other commits from it, or you could close that PR and make a new one from a clean branch that is up to date with the curriculum repo.

If you have access to The Odin Project's Discord you can also ask for help in the #git-help channel and someone might be able to help walk you through those steps. 

Git can be pretty tricky! But this could also be a good learning opportunity 🙂

Reply:
@dm-murphy 

> some prior commits attached

I branched out from my local outdated main which resulted on unexpected side effects. Thanks for pointing that out.

> Git can be pretty tricky! But this could also be a good learning opportunity

Yes, definitely it was a tricky one.

I was able to fix it by hard resetting the branch to upstream and force push to my remote branch. The PR got closed automatically after that, so I made a new one https://github.com/TheOdinProject/curriculum/pull/24478 with only the img file changed.

Reply:
@dm-murphy

I updated the statically image link and cleaned outdated commits related to the image file.